sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/unresolved.scala

```diff
@@ -40,9 +41,12 @@ class UnresolvedException[TreeType <: TreeNode[_]](tree: TreeType, function: Str
  * Holds the name of a relation that has yet to be looked up in a catalog.
  *
  * @param multipartIdentifier table name
+ * @param options options to scan this relation. Only applicable to v2 table scan.
```

Comment:
okay, I just noticed https://github.com/apache/spark/commit/5e825482d70e13a8cb16f1fbdac8139710482d17 added the merging behaviour.

Okay, maybe we should fix the comments here.

Reply:
I will have a followup to fix the comment.

Reply:
opened a PR https://github.com/apache/spark/pull/34075

Reply:
Oh, I create a PR without seeing your comment. Thanks @huaxingao.